ha/core/support_bundle/ha_bundle.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+
+from cortx.utils.schema.payload import Tar
+from cortx.utils.process import SimpleProcess
+from cortx.utils.log import Log
+
+from ha import const
+from ha.core.error import SupportBundleError
+
+class HABundle:
+
+    def __init__(self):
+        """
+        This Class generates the Support Bundle for Component HA.
+        This method will generate bundle for CSM and include the logs in it.
```

Comment:
Is this copy/paste stuff?

Reply:
Yes got copy/paste Corrected in next push

Fixed